v2/cmd/wails/internal/commands/build/build.go

```diff
@@ -162,7 +162,24 @@ func AddBuildSubcommand(app *clir.Cli, w io.Writer) {
 			mode = build.Debug
 			modeString = "Debug"
 		}
+		splitPlatform := strings.Split(platform, "/")
+		targetPlatform := os.Getenv("GOOS")
+		targetArch := os.Getenv("GOARCH")
+		if targetPlatform == "" {
+			targetPlatform = splitPlatform[0]
+		}
+		if targetArch == "" {
+			targetArch = runtime.GOOS
+			if len(splitPlatform) > 1 {
+				targetArch = splitPlatform[1]
+			} else {
+				if system.IsAppleSilicon {
+					targetArch = "arm64"
+				}
+			}
+		}
 
+		platform := targetPlatform + "/" + targetArch
```

Comment:
Doesn't this conflict with the multi target building where we could specify multiple platforms separated by `,`? On Line 184 we split the platform by `,` but since we rebuild the platform here we loose all additional defined platforms.

Reply:
Yes, you do, but I'm going to assume the person compiling is overriding the platform or arch in this way for a reason. We could print a warning?

Reply:
Yeah but it seems like even when not overriding with thr env vars, one can't build multiple platforms anymore.

Reply:
Yeah, that's an issue, but I'm suggesting reverting all of this and just setting the default platform right at the start to be `$GOOS/$GOARCH` if specified or `runtime.GOOS/runtime.GOARCH` if not.

Reply:
That sounds great 👍